software/test_project/src/custom_objects/Avatar.java

```diff
@@ -77,11 +76,6 @@ private boolean collidingWithMapElement(){
 				&& (toMoveY+getHeight() >= element.getY() 
 				&& toMoveY <= (element.getY()+element.getHeight()))
 				)
```

Comment:
commented out code is ok in temporal commits or when it is used for explaining but should not be pushed as final version.